src/itemloader.h

```diff
@@ -149,6 +150,7 @@ enum itemattrib_t {
 	ITEM_ATTR_WRITEABLE3, //deprecated
 
 	ITEM_ATTR_WAREID,
+	ITEM_ATTR_CLASSLEVEL,
```

Comment:
"CLASSLEVEL" feels redundant, it's "classification", no need for "LEVEL" suffix.
It's class and tier.